doc/man7/des_modes.pod

```diff
@@ -16,7 +16,7 @@ other things.
 
 Normally, this is found as the function I<algorithm>_ecb_encrypt().
 
-=over 2
+=over 4
```

Comment:
If we're going to make any change of bulleted lists, I would prefer `=over 2`.  it doesn't have to be in this PR.